beancount/query/query_compile_test.py

```diff
@@ -186,6 +186,14 @@ def test_compile_EvalSub(self):
         c_plus = qc.EvalSub(qc.EvalConstant(17), qc.EvalConstant(18))
         self.assertEqual(Decimal, c_plus.dtype)
 
+    def test_compile_EvalAddAmount(self):
+        c_plus = qc.EvalAdd(qc.EvalAmount(Decimal('1.11','USD')), qc.EvalAmount(Decimal('3.55','USD')))
```

Comment:
This cannot work, the second arguments to `Decimal` are wrong.

Reply:
Sorry, I can the unit test but didn't realize I can it on my installed version of beancounter (2,3.4) and not the source I was editing. My latest commit fixes all the errors I introduced in the unit test.